src/util/string.h

```diff
@@ -84,6 +84,16 @@ inline QString convertWCStringToQString(
     return QString::fromWCharArray(wcs, static_cast<int>(wcsnlen_s(wcs, maxLen)));
 }
 
+/// Remove trailing spaces from the specified string.
+/// Currently only handles simple ASCII spaces (" ").
+inline QString removeTrailingWhitespaces(const QString& str) {
+    QString out = str;
+    while (out.endsWith(' ')) {
+        out.chop(1);
+    }
+    return out;
+}
```

Comment:
Le's fix it here: 

```suggestion
/// Remove trailing spaces from the specified string.
inline QString removeTrailingWhitespaces(QString str) {
    auto it = str.crbegin();
    while (it != str.crend() && it->isSpace()) {
        ++it;
    }
    if (it != str.crbegin()) {
       str.resize(std::distance(it, str.crend()));
    }
    return str;
}
```

Reply:
Did you consider my suggestion?

Reply:
Ah sorry, I was focused on the functional changes in the dialogs.

Looks good, thank you!